src/test/kotlin/org/wfanet/measurement/common/grpc/TransportSecurityTest.kt

```diff
@@ -65,31 +66,50 @@ class TransportSecurityTest {
   @get:Rule val grpcCleanup = GrpcCleanupRule()
 
   private fun startServer(clientAuth: ClientAuth): Server {
+
     val server =
       grpcCleanup
         .register(
           NettyServerBuilder.forPort(0)
-            .sslContext(serverCerts.toServerTlsContext(clientAuth))
-            .addService(healthStatusManager.healthService.withVerboseLogging())
-            .build()
+              .sslContext(serverCerts.toServerTlsContext(clientAuth))
+              .addService(healthStatusManager.healthService.withVerboseLogging())
+                .build()
         )
         .start()
     healthStatusManager.setStatus(SERVICE, HealthCheckResponse.ServingStatus.SERVING)
 
     return server
   }
 
+  private fun startCommonServer(clientAuth: ClientAuth): Server {
```

Comment:
To give some context for some of the changes here:

I added this so I can have a test that includes the CommonServer code. I want to run the code in the tests that creates the server (calls NettyServerBuilder) instead of having separate test code that creates the server. 

As a result, I needed to add a public constructor for Flags (couldn't find a way to instantiate that otherwise).
Also as a result: the port is hardcoded but that shouldn't be a problem? Server is torn down after each test.